inductiva/inductiva
# Add stream logs to prod

This PR makes the minor changes to adequately introduce the streaming of logs into Production mode.

Results:
<img width="740" alt="Screenshot 2024-01-26 at 11 04 07" src="https://github.com/inductiva/inductiva/assets/104431973/1641bbf6-4e61-490e-a4c8-93f20f8d0fc6">


## QA

Setup your environment to use the dev api. Export the env variable INDUCTIVA_TASK_LOGS_URL=wss://logs-dev.inductiva.ai.
Launch a simulation, retrieve the task ID and in another terminal check inductiva logs <task_id> is working correctly.
